### PR TITLE
Updates dependencies

### PR DESCRIPTION
**Pull request by JeroenKnoops**
` yarn upgrade-interactive`

All packages except the reds.

### CODE REVIEW

File: workspaces/demo-app/setup-tests.js [hunk 1]
```diff
@@ -4,4 +4,4 @@ import { createSerializer } from 'jest-emotion'
 import * as emotion from 'emotion'
 
 expect.addSnapshotSerializer(createSerializer(emotion))
-jest.setTimeout(7000)
+jest.setTimeout(9000)
```
Comment (markspanbroek) [inline workspaces/demo-app/setup-tests.js]: Did you mean to commit this change? It doesn't seem to belong in this pull request..

File: workspaces/cogito-ios-app-distribution/src/app/__snapshots__/Main.test.js.snap [hunk 2]
```diff
@@ -1,26 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`renders correctly 1`] = `
-.emotion-2 {
```
Comment (markspanbroek) [inline workspaces/cogito-ios-app-distribution/src/app/__snapshots__/Main.test.js.snap]: Do you know why the dependency update changes the snapshots?

File: workspaces/demo-app/setup-tests.js (hunk 1 above)
Comment (JeroenKnoops) [reply workspaces/demo-app/setup-tests.js]: Locally I needed this, but indeed... this should not have been committed!

File: workspaces/cogito-ios-app-distribution/src/app/__snapshots__/Main.test.js.snap (hunk 2 above)
Comment (JeroenKnoops) [reply workspaces/cogito-ios-app-distribution/src/app/__snapshots__/Main.test.js.snap]: `frontend-react-developer` is updated and forces `emotion` to upgrade from 9.2.12 to 10.0.0. I think this is the reason for the different snapshots.

Comment (markspanbroek) [reply workspaces/cogito-ios-app-distribution/src/app/__snapshots__/Main.test.js.snap]: 👍
